### PR TITLE
HDDS-3885. Create Datanode home page

### PR DESCRIPTION
**Pull request by maobaolong**
## What changes were proposed in this pull request?

We need a Datanode home page. After this, we can do more effort to see the starttime version, build info and more information about hddsdatanode service.

## What is the link to the Apache JIRA

HDDS-3885

## How was this patch tested?

start datanode and see the root page.

http://localhost:9882/

image

### CODE REVIEW

**Comment by maobaolong** [comment]
@adoroszlai Please take a look at this PR, thanks.

**Comment by maobaolong** [comment]
@adoroszlai Thanks for your review. AFAIK, the reason why Rpc Metrics page is empty for datanode is that datanode has no rpc service.PTAL.

**Comment by adoroszlai** [comment]
Thanks @maobaolong for updating the patch.  Let's wait for others to take a look.

**Comment by maobaolong** [comment]
@adoroszlai  Thank you for your review.

**Comment by maobaolong** [comment]
@xiaoyuyao Please take a look at this PR, if we take this PR into 0.6.0, it would be better, thanks.

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java [hunk 1]
```diff
@@ -22,13 +22,11 @@
 import java.net.InetAddress;
 import java.security.KeyPair;
 import java.security.cert.CertificateException;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```
Comment (ChenSammi) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java:25]: Import * is not recommended.

Comment (maobaolong) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java]: Done.

**Comment by maobaolong** [comment]
@ChenSammi Thank you for correct me, PTAL.

**Comment by ChenSammi** [comment]
Tested it locally.  +1.  Thanks @maobaolong for the contribution.

**Pull request merged**